en/installation.md

```diff
@@ -107,10 +107,14 @@ Some users with macOS Sierra have reported freezes when using JabRef. It seems t
 
 #### Some characters are not displayed in the main table (math characters or some upper-cased letter)
 
-This is one the one hand a font problem and seond a lognstanding [Java(fx) bug](https://bugs.openjdk.java.net/browse/JDK-8176835) This might be a problem related to the font you are using. You can download some other font that supports mathematical alphanumeric symbols, for example, FreeSerif or Cambria Math. A list of fonts supporting Math Unicode blocks is available at [http://www.fileformat.info/info/unicode/block/mathematical\_alphanumeric\_symbols/fontsupport.htm](http://www.fileformat.info/info/unicode/block/mathematical\_alphanumeric\_symbols/fontsupport.htm).
+This is one the one hand a font problem and second a lognstanding [JavaFX bug](https://bugs.openjdk.java.net/browse/JDK-8176835). This might be a problem related to the font you are using. You can download some other font that supports mathematical alphanumeric symbols, for example, FreeSerif or Cambria Math. A list of fonts supporting Math Unicode blocks is available at [http://www.fileformat.info/info/unicode/block/mathematical\_alphanumeric\_symbols/fontsupport.htm](http://www.fileformat.info/info/unicode/block/mathematical\_alphanumeric\_symbols/fontsupport.htm).
 {% endtab %}
 {% endtabs %}
 
+#### Submenus from the menu bar close immediately after left click is let go of if the menu bar was clicked in its top half
```

Comment:
This should be moved up to the linux tab, right?

Reply:
Yes, it should be in the Linux tab. I didn't realize there were multiple tabs.